udata/tests/dataset/test_dataset_model.py

```diff
@@ -5,13 +5,14 @@
 from mongoengine import post_save
 
 from udata.models import db, Dataset
+from udata.core.dataset.factories import (ResourceFactory, DatasetFactory,
```

Comment:
I chose consciously the first option as:
- it is more readable to me when it's blocked with other `from udata.core… import …` lines
- it saves a line

If you still prefer the second version I change it at the several places where this is applied.